README.md

```diff
@@ -1,2 +1,64 @@
-# supervision-success-component
+# Supervision Success
+
 A React component powering a miniature web app for exploring the impact of success in supervision on the justice system at large
+
+## Contents
+
+1. [Input File Format](w#input-file-format)
+
+## Input File Format
+
+An example of a valid CSV file can be found in [`params.csv`](/src/params.csv).
+
+The first row of the file (the headers) contains the required revocation model parameters, **in any order**.
+
+- `state`
```

Comment:
I just noticed that we have not yet made the year field dynamic and are still hard-coding to 2017. However, we now have the `Data Year` column in the Parameters sheet, so we should update our spec to include that column and use it to populate the year field in the component.